complaints/templates/complaints/base.html

```diff
@@ -27,4 +27,6 @@
       <a class="navbar-brand" href="#">CCIT BPHC</a>
     </div>
     <div class="collapse navbar-collapse" id="myNavbar">
-    <ul class="nav navbar-nav">
\ No newline at end of file
+    <ul class="nav navbar-nav">
+	{% block content %}
+	{%endblock%}
```

Comment:
<html>
<body>

<h2>HTML Forms</h2>

<form action="/action_page.php">
  First name:<br>
  <input type="text" name="firstname" value="Mickey">
  <br>
  Last name:<br>
  <input type="text" name="lastname" value="Mouse">
  <br><br>
  <input type="submit" value="Submit">
</form> 

<p>If you click the "Submit" button, the form-data will be sent to a page called "/action_page.php".</p>

</body>
</html>